test/e2e/scenarios/scenario1.yaml

```diff
@@ -0,0 +1,40 @@
+id: "RG_CMD_basic_create_update"
+description: "In this test we create RG with absolutely minimum configuration and try to update it."
+usecases:
+  - replication_group
+  - cmd
+#marks:
+#  - slow
+#  - blocked
+customResourceReference:
+  apiVersion: $CRD_GROUP/$CRD_VERSION
+  kind: ReplicationGroup
+  metadata:
+    name: test$RANDOM_SUFFIX
+steps:
+  - id: "create_CMD_replication_group"
+    description: "Initial config"
+    create:
+      spec:
+        engine: redis
+        replicationGroupID: test$RANDOM_SUFFIX
+        replicationGroupDescription: Basic create and update of CMD RG
+        cacheNodeType: cache.t3.micro
+        numNodeGroups: 1
```

Comment:
This is cool to allow `spec` values directly in the scenario YAML definition. I'd recommend also allowing to load from a filepath as wel.

Reply:
Updated code. `create`, `patch`, `delete` now support reading custom resource from filepath. 

Filepath is resolved by taking `resource_directory` parameter into account:
```
loader.load_scenario(scenario_file_path, resource_directory, input_replacements)
```

Example:
```
steps:
  - id: "create_CMD_replication_group"
    description: "Initial config"
    create: "replicationgroup_cmd.yaml"
```